core/amber/src/main/resources/python_udf/topic_modeling_mallet_trainer.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+# Use gensim 3.8.3
+import gensim
+import gensim.corpora as corpora
+import pandas
+import os
+
+import texera_udf_operator_base
+
+
+class TopicModeling(texera_udf_operator_base.TexeraBlockingUnsupervisedTrainerOperator):
+    logger = logging.getLogger("PythonUDF.TopicModelingTrainer")
+
+    @texera_udf_operator_base.exception(logger)
```

Comment:
I think this is an old way of doing logs, which is in the middle version of #1119 (but this is different from the current master). I think it won't work on the current master code.

Reply:
I tried to test this myself, but I might not have mallet env configured correctly. Can you test it one more time with the latest master?

Reply:
will remove and test